server/db/level/jobs.coffee

```diff
@@ -8,9 +8,9 @@ module.exports =
   # taking two arguments; a payload and an error logger function
   initQueue: (jobName, worker, maxConcurrency)->
     db = levelBase.sub "job:#{jobName}"
-
-    # If runJobsInQueue is true, push and run jobs
-    if CONFIG.runJobsInQueue
+    # Only starts job marked in runJobsInQueue, push and run jobs
```

Comment:
proposition of rephrasing: `# If the job is whitelisted, push and run jobs`

Reply:
is it a whtelist tho, its a key value configuration not a hardcoded stored array.
maybe `flagged` instead of `marked`

Reply:
maybe `flagged`, but the beginning of the phrase still doesn't make sense with the `push and run jobs` part, no?